packages/now-next/src/index.ts

```diff
@@ -845,6 +845,7 @@ export const build = async ({
       // Next.js page lambdas, `static/` folder, reserved assets, and `public/`
       // folder
       { handle: 'filesystem' },
+      { src: '/_next(?:/.*)?', status: 404, dest: '/_error' },
```

Comment:
Shouldn't this be moved to before `{ handle: 'filesystem' }` and just be `{ src: '/_next(.*)' }`?

Reply:
Actually, there's dynamic data routes which exist under the `/_next` path. We can't route this to 404.

Reply:
No because if it 404s before `{ handle: 'filesystem' }` it will still continue to route

Reply:
What route?

Reply:
`dynamicDataRoutes`

Reply:
I'm asking what the path pattern is for those (e.g. `/_next/dynamic/whatever`)

Reply:
Yeah, GitHub only showed your question after I already posted that.

It looks like this:
```
`^\\/_next\\/data\\/${escapedBuildId}\\/blog\\/([^\\/]+?)\\.json$`
```